pyproject.toml

```
@@ -21,4 +21,59 @@ select = [
 split-on-trailing-comma = false
 
 [build-system]
-requires = ["setuptools<64", "wheel"]
+requires = ["setuptools>=68,<69", "wheel"]
```

The [recommendation](https://setuptools.pypa.io/en/latest/userguide/quickstart.html#basic-use) now is to use:

```suggestion
requires = ["setuptools"]
```